.changeset/swift-dryers-invent.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
I think in this case we can just override the bot

Reply:
This is actually what the bot says to do for this case in the CI output: `npx changeset --empty`